lib/Runtime/Library/JSONStack.h

```diff
@@ -15,6 +15,7 @@ namespace JSON
     class JSONStack
     {
     private:
+        Js::Var        tempValues[2];
```

Comment:
@digitalinfinity not sure about the answer. I added a destructor to make sure they are nullified.